plugin/dubbo/route.go

```diff
@@ -79,22 +79,73 @@ func buildRoute(context *model.EnvoyFilterContext) []*dubbo.Route {
 			routeAction = buildSingleCluster(http, service)
 		}
 
-		routes = append(routes, &dubbo.Route{
-			//todo: convert virtual service HTTP Route Match to Dubbo Route Match
-			Match: &dubbo.RouteMatch{
-				Method: &dubbo.MethodMatch{
-					Name: &matcher.StringMatcher{
-						MatchPattern: &matcher.StringMatcher_SafeRegex{
-							SafeRegex: &matcher.RegexMatcher{
-								EngineType: regexEngine,
-								Regex:      ".*",
+		dubboRoute := new(dubbo.Route)
+		for _, m := range http.Match {
+			method := m.Method
```

Comment:
I suggest we only allow one method match in a dubbo route, otherwise, the handling of the combination of method and headers can become complex. If we do need different routes for multiple methods, we can handle that at the upper level (multiple routes). Just like the dubbo envoypoxy API does:
https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/dubbo_proxy/v3/route.proto#envoy-v3-api-msg-extensions-filters-network-dubbo-proxy-v3-route

Reply:
One to one mapping of VirtualSerive HTTPRoute and DubboRoute is clearer for me.

Reply:
ok